starter_templates/haskell/package.yaml

```diff
@@ -12,10 +12,15 @@ license:             BSD3
 
 dependencies:
 - base >= 4.7 && < 5
-- network
-- bytestring
-- attoparsec # can help when implementing the RESP parser
-- time # useful in the expiry stage
+- network-simple  # establish a simple TCP network
+- bytestring  # useful to send bytes over the network
+- network  # required when using bytestring
+- megaparsec  # may be useful for parsing the input
+- parser-combinators  # contains a helpful count function
```

Comment:
Megaparsec is a fine library but can we keep Attoparsec, too? Megaparsec is a totally different library and it's harder to use. Furthermore, attoparsec is more standalone b/c it comes with `count` so `parser-combinators` dependency isn't used.

Reply:
@jzwood thanks for your review. We decided for Megaparsec in our solution write-up as we felt it is a bit more convenient to use for a beginner (at least how we used it), but good point re `count`, I was not aware of this. However, one can implement the solution in either parser library - feel free to submit a PR to re-add attoparsec to the starter template.